includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -84,6 +84,12 @@ private function adjust_and_replace_node( $node ) {
 		$old_attributes = AMP_DOM_Utils::get_node_attributes_as_assoc_array( $node );
 		$new_attributes = $this->filter_attributes( $old_attributes );
 		$new_attributes = $this->enforce_sizes_attribute( $new_attributes );
+		if ( $new_attributes['height'] === '' ) {
+			$new_attributes['height'] = self::FALLBACK_HEIGHT;
+		}
+		if ( $new_attributes['width'] === '' ) {
```

Comment:
Instead of strict equality with an empty string, perhaps `empty()` just in case the value is `null` or `false`.